src/functions/template-tags/link.php

```diff
@@ -320,13 +320,18 @@ function tribe_get_listview_prev_link( $term = null ) {
 }
 
 /**
- * Link to next List View
+ * Retrieves the URL for the next page in the list view.
  *
- * Returns a link to the next list view page
+ * This function returns a link to the next page of events in the list view,
+ * optionally filtered by a specific term (category or taxonomy).
```

Comment:
```suggestion
 * optionally filtered by a specific taxonomy's term.
```

Category is a taxonomy.